main/main.cpp

```diff
@@ -232,6 +232,14 @@ static String validate_extension_api_file;
 #endif
 bool profile_gpu = false;
 
+#define BLOCK_DEVICE(m_vendor, m_name)      \
+	{                                       \
+		Dictionary device;                  \
+		device["vendor"] = m_vendor;        \
+		device["name"] = m_name;            \
+		device_blocklist.push_back(device); \
+	}
```

Comment:
I'd move that macro just before the place where it's used, and `#undef` it afterwards. That would make things cleaner IMO.

Reply:
Done.